ui/keyboardMove/src/plugins/keyboardMove.ts

```diff
@@ -31,6 +33,9 @@ export default (opts: Opts) => {
     if (!submitOpts.isTrusted) return;
     // consider 0's as O's for castling
     v = v.replace(/0/g, 'O');
+    if (v.match(iccfRegex) || v.match(partialIccfRegex)) {
```

Comment:
For simplicity, I think we can just combine these regexes into one. Otherwise, looks good to me 👍